acceptance/api/v1/application_export_test.go

```diff
@@ -60,8 +61,8 @@ var _ = Describe("AppPart Endpoint", func() {
   stageID: ""
   start: null
   tlsIssuer: epinio-ca
-  username: admin
-`, app)))
```

Comment:
Maybe this deserves a comment on why `base58` was chosen (over base64 for example)